test/battle/move_effect/attack_accuracy_up.c

```diff
@@ -1,4 +1,18 @@
 #include "global.h"
 #include "test/battle.h"
 
-TO_DO_BATTLE_TEST("Hone Claws increases Attack and Accuracy by one stage each");
+SINGLE_BATTLE_TEST("Hone Claws increases Attack and Accuracy by one stage each")
```

Comment:
Hmm, thinking about this, we should probably split this test into two, since for the most part, Attack tests are checked via damage multipliers and accuracy ones are done with `PASSES_RANDOMLY`

Reply:
Does this mean you want two additional tests written? This doesn't test for damage multipliers OR accuracy in using moves, it just checks for the stat buffs, and assumes that the stat buffs themselves work, as a tonne of stuff should fail if they don't.

Reply:
I guess it does test for the messages